app/components/layout/Navigation/MyBuilds/index.js

```diff
@@ -82,9 +82,15 @@ class MyBuilds extends React.Component {
         <DropdownButton className={classNames("flex-none py0", { "lime": this.state.isDropdownVisible })} onMouseEnter={this.handleButtonMouseEnter}>
           {'My Builds '}
           <div className="xs-hide">
-            <CSSTransitionGroup transitionName="transition-appear-pop" transitionEnterTimeout={200} transitionLeaveTimeout={200}>
+            <CSSTransition
```

Comment:
I wonder if there's a `eslint` rule we can have that forces this as the style. I personally prefer the former, but I'm happy to standardise on the latter.

Reply:
Last time I checked, none existed that didn’t mandate that you always do one or the other. I think inline makes sense for one or two short props, but as soon as it gets longer than like, 3x the component name I much prefer multiline. Makes diffs cleaner IMO.